### DISCUSSION TITLE
LazyVim says: Lazyvim Needs Neovim 0.9.0 but i have this version

### DISCUSSION THREAD

**Discussion by lucas11222** [LazyVim/LazyVim#2037] [Q&A] [answered]
im reinstaled and same

**Comment by dimas-cyriaco** [comment]
Without more context it's hard to help you. What error you are getting? Which version of neovim you have?

**Reply by lucas11222** [reply]
0.9.0. I already solved it but I have a problem, it only works when I open it in the taskbar and when I put the "nvim" command it says:
LazyVim Requires Neovim 0.9.0. I reinstalled the neovim and the same thing, and the strangest thing is that it still says that just as uninstalled. NOTE: Restart and same

**Reply by dimas-cyriaco** [reply]
Maybe you have multiple instalations of neovim. You could try uninstalling and running `which nvim` to see if there's other version intalled. Which OS are you using, and how are you installing neovim?

**Reply by lucas11222** [reply]
im on windows not linux EDIT: i found it and uninstaled but the command is not working

**Reply by dimas-cyriaco** [reply]
Oh, sorry. I don't think I'll be able to help you much, then... Maybe try to install neovim using a package manager like [Chocolatey](https://community.chocolatey.org/packages/neovim).

**Reply by dpetka2001** [reply, marked as answer]
I used to install Neovim by the Ubuntu apt repository and then build from source. I ended up transitioning to [bob](https://github.com/MordechaiHadad/bob) and it's really great. You can easily switch between stable and nightly version. I highly recommend it for installing Neovim, since it's also a multi-platform tool (although I only have experience with it on Linux). Also, after some googling, I found out that the equivalent of `which` command on Linux is `where` on Windows.

**Reply by lucas11222** [reply]
Ok. I found it, i have to add in the PATH from the cmd
